components/org.wso2.carbon.identity.api.server.organization.user.invitation.management/org.wso2.carbon.identity.api.server.organization.user.invitation.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/user/invitation/management/common/SharedUserInvitationMgtConstants.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.organization.user.invitation.management.common;
+
+/**
+ * Holds the constants which the shared user invitation management API component is using.
+ */
+public class SharedUserInvitationMgtConstants {
+
+    public static final String ERROR_PREFIX = "OUI-";
+
+    /**
+     * Enum for shared user invitation management related errors.
+     * Error Code - code to identify the error.
+     * Error Message - What went wrong.
+     * Error Description - Why it went wrong.
+     */
+    public enum ErrorMessage {
+        ERROR_CODE_USER_NOT_FOUND("60000",
+                "Invalid user identification provided.",
+                "Invalid user identification %s is provided."),
+        ERROR_CODE_CREATE_INVITATION("60001",
```

Comment:
Isn't this an internal server error?
Usually, we use 650XX for server errors.
separate out client errors and server errors.
Check others and categorize accordingly

Reply:
Categorized the client and server errors